### PR TITLE
Change keystone middleware auth_token to use identity_uri

### PR DESCRIPTION
**Pull request by mjura**
We would like to update middleware for keystone, because:

```
2015-01-26 08:48:30.131 24606 WARNING keystonemiddleware.auth_token [-] Configuring admin URI using auth fragments. This
 is deprecated, use 'identity_uri' instead.
2015-01-26 08:48:30.132 24606 WARNING keystonemiddleware.auth_token [-] Configuring auth_uri to point to the public iden
tity endpoint is required; clients may not be able to authenticate against an admin endpoint
```

### CODE REVIEW

**Comment by tboerger** [comment]
LGTM

File: chef/cookbooks/openstack-database/templates/default/api-paste.ini.erb [hunk 1]
```diff
@@ -9,23 +9,14 @@ use = call:trove.common.wsgi:versioned_urlmap
...
-paste.filter_factory = keystoneclient.middleware.auth_token:filter_factory
-auth_uri = <%= @identity_uri.to_s %>
-auth_host = <%= @identity_admin_uri.host %>
-auth_port = <%= @identity_admin_uri.port %>
-auth_protocol = <%= @identity_admin_uri.scheme %>
-admin_token = <%= @admin_token %>
-# signing_dir is configurable, but the default behavior of the authtoken
-# middleware should be sufficient.  It will create a temporary directory
-# in the home directory for the user the trove process is running as.
-signing_dir = <%= node["openstack"]["database"]["api"]["auth"]["cache_dir"] %>
```
Comment (vuntz) [inline chef/cookbooks/openstack-database/templates/default/api-paste.ini.erb]: Don't we need to put that setting in trove.conf now?

**Comment by rsalevsky** [comment]
@mjura, this repository got merged into [crowbar-openstack](https://github.com/crowbar/crowbar-openstack).
Please open a new pull request there. __Do NOT merge here anymore!__

**Comment by mjura** [comment]
Don't have time to work on it. It can reopened in the future

**Pull request closed without merging**